SimPEG/EM/Static/DC/ProblemDC.py

```diff
@@ -155,9 +237,8 @@ def getA(self):
             # Handling Null space of A
             I, J, V = sp.sparse.find(A[0, :])
             for jj in J:
-                A[0,jj] = 0.
-
-            A[0, 0] = 1./Vol[0]
+                A[0, jj] = 0.
+            A[0, 0] = 1.
```

Comment:
@sgkang: this changes how we are handling the null space. Do you have a test on this? is it better than using the volume?

Reply:
I believe this that volume term is when formulating problem with finite difference. 
For our case, A[0, 0] = 1. is making more sense. And it seems do not have effect on the accuracy.

Reply:
kk, thanks for clarifying